ui/test/nuxeo-date-picker.test.js

```diff
@@ -117,9 +117,9 @@ suite('nuxeo-date-picker', () => {
         element.defaultTime = '14:35:19';
         getInput(element).value = '2003-02-20';
         const localEltValue = moment(element.value).local();
-        expect(localEltValue.hour()).to.be.equal(14);
-        expect(localEltValue.minute()).to.be.equal(35);
-        expect(localEltValue.second()).to.be.equal(19);
+        expect(localEltValue.hour()).to.be.equal(moment(element.value).hour());
+        expect(localEltValue.minute()).to.be.equal(moment(element.value).minute());
+        expect(localEltValue.second()).to.be.equal(moment(element.value).second());
```

Comment:
Not sure what is the goal this change? Now we are just comparing `moment(element.value)` to `moment(element.value)`.

Reply:
Since you are just doing an a11y improvement, why do you need to change the unit test?

Reply:
previously values were hardcoded and after each call value was changed and test cases were failing. that was the reason for failing the test case so I have changed the put dynamic value.

Reply:
But this test is not currently testing anything about our element. Because
```
localEltValue = moment(element.value).local();
```
 then
```
expect(localEltValue.hour()).to.be.equal(moment(element.value).hour());
```
is actually the same as
```
expect(moment(element.value).local()).to.be.equal(moment(element.value).hour());
```
with the only difference between both values being the left one as the `.local()` call.  At most, this is testing `moment` itself and not our element.

Reply:
It makes sense. So I revert this code.